src/components/molecules/Monaco/Monaco.tsx

```diff
@@ -191,7 +188,8 @@ const Monaco = (props: {diffSelectedResource: () => void; applySelection: () =>
     if (selectedResourceId) {
       // this will slow things down if document gets large - need to find a better solution...
       const documents = parseAllDocuments(newValue);
-      setValid(documents.length > 0 && !documents.some(d => !isValidResourceDocument(d)));
+      // only accept single document changes for now
```

Comment:
does editing multi-resource files still work in this case? if not, should the editor be read-only?

Reply:
i'll double-check, but yes - selectedResourceId should not be set if the user is editing a multiresource file. This specific case is only to ensure that a user doesn't try to turn a single resources into multiple resources by manually inserting a resource delimiter (which we could support in the future)